Source/WebCore/rendering/style/StyleImageSet.h

```diff
@@ -31,7 +31,7 @@ namespace WebCore {
 class StyleImageSet final : public StyleMultiImage {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    static Ref<StyleImageSet> create(Vector<ImageWithScale>);
+    static Ref<StyleImageSet> create(Vector<ImageWithScale>, Vector<size_t>);
```

Comment:
I was a bit torn here and I tried to follow the pattern I saw elsewhere. What I was going for was `create` would take either the vectors by value or you could pass a temporary like `StyleImageSet::create(WTFMove(v), WTFMove(u))`. Then the constructor will take by `Vector&&` so we can skip a copy by `WTFMove`-ing in the implementation of `create`.

Since the construction site of this object is only in once place in `CSSImageSetValue` and we're taking ownership of the vectors created there it's fine to make the create function take the arguments by rvalue reference I think.